### PR TITLE
Add externs for countReset and timeLog

### PR DESCRIPTION
**Pull request by realityforge**

### CODE REVIEW

**Comment by realityforge** [comment]
Note this has been extracted from #3268 to simplify merging

File: externs/browser/webkit_dom.js [hunk 1]
```diff
@@ -194,6 +200,12 @@ Console.prototype.profileEnd = function(opt_title) {};
  */
 Console.prototype.time = function(name) {};
 
+/**
+ * @param {string} label
+ * @return {undefined}
+ */
+Console.prototype.timeLog = function(label) {};
```
Comment (tjgq) [inline externs/browser/webkit_dom.js]: Per the spec, this needs an additional `@param {*...} data`.

https://console.spec.whatwg.org/#timing

Comment (realityforge) [reply externs/browser/webkit_dom.js]: Interesting - I had not seen that before. It is not documented at https://developer.mozilla.org/en-US/docs/Web/API/Console/timeLog but I will try it out in the browser and if it works add it in.

Comment (realityforge) [reply externs/browser/webkit_dom.js]: New commit pushed with change

**Comment by tjgq** [comment]
Created Google internal issue b/128865976. Imported for internal review and testing.

**Pull request closed without merging**